### DISCUSSION TITLE
Scrips Folder not showing

### DISCUSSION THREAD

**Discussion by tuklu** [chrisant996/clink#465] [Q&A] [answered]
![image](https://github.com/chrisant996/clink/assets/85573517/72786a5a-eee9-4e5a-8dc8-0e6a5b619650)

**Comment by chrisant996** [comment, marked as answer]
Please explain what you're trying to ask or report.

The screen shot shows the Clink program files.
There should not be a scripts folder there.

I recommend reading the documentation, for example:
- [Getting Started](https://chrisant996.github.io/clink/clink.html#getting-started)
- [File Locations](https://chrisant996.github.io/clink/clink.html#file-locations)
- [Extending Clink With Lua](https://chrisant996.github.io/clink/clink.html#extending-clink)
- [Location of Lua Scripts](https://chrisant996.github.io/clink/clink.html#lua-scripts-location)
